gcs/README.md

```diff
@@ -0,0 +1,246 @@
+# Google Cloud Storage Tasks
+
+These `Tasks` are for copying to and from GCS buckets from Pipelines.
+
+These `Tasks` do a similar job to the `GCS` `PipelineResource` and
+are intended as its replacement. This is part of our plan to [offer replacement
+`Tasks` for Pipeline Resources](https://github.com/tektoncd/catalog/issues/95)
+as well as
+[document those replacements](https://github.com/tektoncd/pipeline/issues/1369).
+
+## `gcs-download`
+
+A `Task` that fetches files or directories from a GCS bucket and puts them
+on a Workspace.
+
+### Workspaces
+
+* **credentials**: A workspace that contains a service account key as a JSON file.
+    This workspace should be populated from a Secret in your TaskRuns and PipelineRuns.
+* **output**: A workspace for this Task to copy the files from GCS in to.
+
+### Parameters
+
+* **name**: The name of the file or directory that will be written to the output workspace. (_required_)
+* **location**: The address, including "gs://", of the bucket you'd like to copy from. (_required_)
+* **typeDir**: Set this to "true" if the object you are copying is a directory. (_default_: "false")
```

Comment:
note we might wanna revisit `typeDir` as it's a bit confusing: https://github.com/tektoncd/pipeline/issues/1234